workbench_access.install

```diff
@@ -44,3 +46,24 @@ function workbench_access_update_8001() {
   $config->save(TRUE);
 }
 
+/**
+ * Update to new access-scheme config entity.
+ */
+function workbench_access_update_8002() {
+  // Stash this for later.
+  \Drupal::state()->set('workbench_access_original_configuration', \Drupal::config('workbench_access.settings'));
+  // Alter it to the new format.
+  $config = \Drupal::configFactory()->getEditable('workbench_access.settings');
+  foreach (['scheme', 'label', 'plural_label', 'fields', 'parents'] as $delete) {
+    $config->delete();
+  }
+  $config->save(TRUE);
+}
+
+/**
+ * Create the new entity type.
+ */
+function workbench_access_update_8003() {
+  // Install our config entity.
+  \Drupal::entityDefinitionUpdateManager()->applyUpdates();
```

Comment:
We don't want to install all the entities available to EDUM. We just want `access_scheme` so I think this would be fine.
```php
  $entity_manager = \Drupal::entityTypeManager();
  $update_manager = \Drupal::entityDefinitionUpdateManager();

  // Install 'registrant_type' config entity.
  $entity_manager->clearCachedDefinitions();
  $type = $entity_manager->getDefinition('access_scheme');
  $update_manager->installEntityType($type);
```

Reply:
fixed, thanks